src/node.cc

```diff
@@ -293,10 +293,13 @@ static struct {
                     "so event tracing is not available.\n");
   }
   void StopTracingAgent() {}
+#endif  // !NODE_USE_V8_PLATFORM
+
+#if NODE_USE_V8_PLATFORM == 0 || HAVE_INSPECTOR == 0
```

Comment:
> sorry to nag, but what about #if !NODE_USE_V8_PLATFORM || !HAVE_INSPECTOR

Yes, that works as it is equivalent to the current statement. I'm fine with either.